backend/api/Utilities/PagedList.cs

```diff
@@ -33,14 +33,12 @@ public PagedList(List<T> items, int currentPage, int pageSize, int totalCount)
             int pageSize
         )
         {
-            // Adding order by constant value 1, which does not change order of rows
-            // but stops the runtime warning
             int totalCount = await source.CountAsync();
             List<T> items;
             if (source.Expression.Type is IOrderedQueryable)
                 items = await source.Skip((pageNumber - 1) * pageSize).Take(pageSize).ToListAsync();
             else
-                items = await source.OrderBy(x => 1).Skip((pageNumber - 1) * pageSize).Take(pageSize).ToListAsync();
+                items = await source.Skip((pageNumber - 1) * pageSize).Take(pageSize).ToListAsync();
```

Comment:
we might as well remove the if-else if we make this change I guess

Reply:
I'm not quite sure when this else statement is reached for mission runs though, since in the ReadAll method we always return an OrderedQueryable. I think that's something we need to investigate. It seems like the only time we would reach this else statement is when we fetch Source objects.